### PR TITLE
upgrade option in pkg_install

### LINKED ISSUE #42: pkg_install upgrade option
Now that `pkgdepends` supports it.

### PR DESCRIPTION
**Pull request by gaborcsardi**
Closes #42.

### CODE REVIEW

**Comment by gaborcsardi** [comment]
Actually, the slowness has nothing to do with `upgrade` mode, so you might as well merge it.....

File: R/package.R [hunk 1]
```diff
@@ -2,16 +2,20 @@
...
 #' @importFrom crayon blue
 #' @export
-pkg_install <- function(pkg, lib = .libPaths()[[1L]], num_workers = 1L) {
+pkg_install <- function(pkg, lib = .libPaths()[[1L]], upgrade = FALSE,
+                        num_workers = 1L) {
   r <- remotes$new(pkg, library = lib)
 
   # Solve the dependency graph
-  r$solve()
+  policy <- if (upgrade) "upgrade" else "lazy"
```
Comment (jimhester) [inline R/package.R]: Maybe make this `isTRUE(update)`, just so we don't need to worry about non-logical inputs?

Comment (gaborcsardi) [reply R/package.R]: Yeah, thought about that. I'll add argument checks to all public functions at some point, that's why I didn't do it. #45

Comment (jimhester) [reply R/package.R]: Ok

**Pull request merged**